src/main/java/com/epam/third/task/entities/ObservableSphere.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.third.task.entities;
+
+import com.epam.third.task.observer.Observer;
+import com.epam.third.task.observer.SphereStateChangeEvent;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ObservableSphere extends SphereWithId {
```

Comment:
could be just one class
ObservableSphere  extends Sphere
and both id and observables inside (for simplicity)